src/components/Layout.js

```diff
@@ -85,4 +85,14 @@ exports.addSystemTestSupport = function(){
         browser.get(url);
         browser.sleep(3000);
     });
+};
+
+exports.acceptAlert = function(){
+    browser.sleep(1000);
+    if(browser.browserName === 'chrome'){
+        browser.switchTo().alert().accept();
+    }
+    if(browser.browserName === 'phantomjs'){
```

Comment:
we might change the browser to any other than phantomjs in the future, I want to make sure this code only runs for phantomjs